i18n/zh-hant.php

```diff
@@ -0,0 +1,347 @@
+                  if (!$("#s1fg3").hasClass("has-error")) {
+                      $("#s1fg3").addClass("has-error");
+                  }
+                  s1vi++;
+              }
+          }
+          if (s1vi == 0) {
+              $("html, body").animate({
+                  scrollTop: $(s2).offset().top
+              }, 400);
+              if ($("#s1fg1").hasClass("has-error")) {
+                  $("#s1fg1").removeClass("has-error");
+              }
+              if ($("#s1fg2").hasClass("has-error")) {
+                  $("#s1fg2").removeClass("has-error");
+              }
+              if ($("#s1fg3").hasClass("has-error")) {
+                  $("#s1fg3").removeClass("has-error");
+              }
+          }
+      }
+      function s4v() {
+          if (!$("#licensei").val().match(/\w/)) {
+              if (!$("#s4fg").hasClass("has-error")) {
+                  $("#s4fg").addClass("has-error");
+              }
+          } else {
+              $("form").submit();
+          }
+      }
+    </script>
+  </head>
+  <body style="background: url('bg.png') no-repeat fixed right bottom; overflow:hidden;">
+
+    <?php
+        $relgen = "0.9.9";
+        date_default_timezone_set("UTC");
+        $starttime = date("H:i:s");
+        $trn = $name = $rep = $auth = $filer = $license = $s1 = $s2 = $s3 = "";
+        
+        if ($_SERVER["REQUEST_METHOD"] == "POST") {
+            $starttime = test_input($_POST["starttime"]);
+            $trn = test_input($_POST["trn"]);
+            $name = test_input($_POST["name"]);
+            $rep = test_input($_POST["rep"]);
+            $auth = test_input($_POST["auth"]);
+            $filer = test_input($_POST["filer"]);
+            $license = test_input($_POST["license"]);
+            $s1 = test_input($_POST["s1"]);
+            $s2 = test_input($_POST["s2"]);
+            $s3 = test_input($_POST["s3"]);
+        }
+        
+        function test_input($data) {
+            $data = trim($data);
+            $data = stripslashes($data);
+            $data = htmlspecialchars($data);
+            return $data;
+        }
+        
+        if ($_POST["result"] == "1") {
+            ?>
+            <script type="text/javascript">
+            $(function() {
+                $("html, body").animate({
+                    scrollTop: $(result).offset().top
+                }, 400);
+            });
+            </script>
+            <?php
+        }
+    ?>
+
+    <div class="container">
+      <h1>維基媒體 OTRS 釋出產生器 <small><a id="meta" tabindex="0" data-toggle="popover" data-placement="bottom" data-trigger="focus" data-content="created and maintained by <a href='//meta.wikimedia.org/wiki/User:FDMS4' target='_blank'>FDMS</a><br />© (<a href='//joinup.ec.europa.eu/sites/default/files/eupl1.1.-licence-en_0.pdf' target='_blank'>EUPL 1.1</a>) <a href='http://fdms.eu' target='_blank'>Stöger Florian D. M.</a" style="color:#777;"><?php echo $relgen;?></a></small></h1>
```

Comment:
Just out of curiosity, why all these `//` protocol-relative links? WMF is using HTTPS anyway...